lib/Controller/SocialApiController.php

```diff
@@ -73,13 +74,20 @@ public function __construct(string $AppName,
 	 * @param {String} type the kind of information interested in
 	 * @returns {array} an array of supported social networks
 	 */
-	public function supported(string $type) : ?array {
-		switch ($type) {
-			case 'avatar':
-				return array('facebook','twitter');
-			default:
-				return array();
+	public function getSupportedNetworks(string $type) : ?array {
+
+		$supported = array();
+		$supported['avatar'] = array('facebook','twitter');
```

Comment:
Those are really what php const are for :)

Reply:
Yes, I have it on my list - don't worry :smile: 
I am just closing one comment after the other, and as long as it's not marked as resolved, I will take care of it. Thought that it's better to push each change to see why I changed things.